admin/server-timing.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Server-Timing API admin integration file.
+ *
+ * @package performance-lab
+ */
+
+/**
+ * Adds the Server-Timing page to the Tools menu.
+ *
+ * @since n.e.x.t
+ */
+function perflab_add_server_timing_page() {
+	$hook_suffix = add_management_page(
+		__( 'Server-Timing', 'performance-lab' ),
+		__( 'Server-Timing', 'performance-lab' ),
+		'manage_options',
+		PERFLAB_SERVER_TIMING_SCREEN,
+		'perflab_render_server_timing_page'
+	);
+
+	// Add the following hooks only if the screen was successfully added.
+	if ( false !== $hook_suffix ) {
+		add_action( "load-{$hook_suffix}", 'perflab_load_server_timing_page' );
+	}
+
+	return $hook_suffix;
+}
+add_action( 'admin_menu', 'perflab_add_server_timing_page' );
+
+/**
+ * Initializes settings sections and fields for the Server-Timing page.
+ *
+ * @since n.e.x.t
+ */
+function perflab_load_server_timing_page() {
+	add_settings_section(
+		'benchmarking',
+		__( 'Benchmarking', 'performance-lab' ),
+		static function() {
+			?>
+			<p>
+				<?php esc_html_e( 'In this section, you can provide hook names to include measurements for them in the Server-Timing header.', 'performance-lab' ); ?>
+				<br>
+				<?php
+				echo wp_kses(
+					__( 'For any hook name provided, the <strong>cumulative duration between all callbacks</strong> attached to the hook is measured, in milliseconds.', 'performance-lab' ),
+					array( 'strong' => array() )
+				);
+				if ( ! perflab_server_timing_use_output_buffer() ) {
+					?>
+					<br>
+					<?php
+					echo wp_kses(
+						__( 'Since the Server-Timing header is sent before the template is loaded, only hooks before the <code>template_include</code> filter can be measured.', 'performance-lab' ),
```

Comment:
I'd use sprintf with a placeholder for `template_include` so that it does not get translated